activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -268,6 +274,46 @@ public void createProcessInstanceAndValidateHappyPath() {
         assertThat(RuntimeTestConfiguration.discardImageConnectorExecuted).isEqualTo(false);
     }
 
+    @Test
+    public void createLinkEventProcessInstanceAndValidateHappyPath() {
+        //when
+        ProcessInstance linkProcess = processRuntime.start(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(LINK_PROCESS)
+            .withVariable("expectedKey",
+                true)
+            .build());
+
+        assertThat(RuntimeTestConfiguration.completedProcesses).contains(linkProcess.getId());
```

Comment:
In addition to checking the process completion, please check that you have a completed event for throw and catch events.